viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1065,6 +1074,37 @@ export class Cognite3DViewer {
     return intersections.length > 0 ? intersections[0] : null;
   }
 
+  /**
+   * Raycasting model(s) for finding where the ray intersects with the model. Note: Only Cad models.
+   * @param offsetX X coordinate in pixels (relative to the domElement).
+   * @param offsetY Y coordinate in pixels (relative to the domElement).
+   * @returns A promise that if there was an intersection then return the intersection point.
+   */
+  async getIntersectionPixelFromBuffer(offsetX: number, offsetY: number): Promise<THREE.Vector3> {
```

Comment:
This should not be a public function in `Cognite3dViewer`. Also, does this mean that point clouds are not supported?

Reply:
@larsmoa any suggestion on how to overcome by not exposing this to the API user? I am running out of idea here